docs/data-sources/dashboard_list.md

```diff
@@ -16,6 +16,23 @@ Use this data source to retrieve information about an existing dashboard list, f
 data "datadog_dashboard_list" "test" {
   name = "My super list"
 }
+
+# Create a dashboard and register it in the list above.
+resource "datadog_dashboard" "time" {
+  title        = "TF Test Layout Dashboard"
+  description  = "Created using the Datadog provider in Terraform"
+  dashboard_lists = ["${data.datadog_dashboard_list.test.id}"]
+  layout_type  = "ordered"
+  is_read_only = true
+  widget {
+    alert_graph_definition {
+      alert_id  = "1234"
+      viz_type  = "timeseries"
+      title     = "Widget Title"
+      live_span = "1h"
+    }
+  }
+}
```

Comment:
```suggestion
```

Thanks for the contribution. The datadog terraform provider docs are auto generated. Could you move the example to https://github.com/DataDog/terraform-provider-datadog/blob/master/examples/data-sources/datadog_dashboard_list/data-source.tf and run `make docs` to auto-generate the docs.